internal/services/kms_import_key_service.go

```diff
@@ -59,9 +60,6 @@ func (s *kmsImportService) ImportDistributionAccountKey(ctx context.Context, dis
 
 	err = s.keypairStore.Insert(ctx, kp.Address(), output.CiphertextBlob)
 	if err != nil {
-		if errors.Is(err, store.ErrPublicKeyAlreadyExists) {
-			return err
-		}
 		return fmt.Errorf("storing distribution account encrypted private key: %w", err)
```

Comment:
I removed this assertion because the function returns the same error anyway, and it can be parsed downstream using `errors.Is`